scripts/linters/js_ts_linter.py

```diff
@@ -171,12 +159,8 @@ def _validate_and_parse_js_and_ts_files(self):
         # Select JS files which need to be checked.
         files_to_check = self.all_filepaths
         parsed_js_and_ts_files = dict()
-        if not self.verbose_mode_enabled:
-            python_utils.PRINT('Validating and parsing JS and TS files ...')
+        concurrent_task_utils.log('Validating and parsing JS and TS files ...')
```

Comment:
This line should be printed when the verbose mode is not enabled in the original code. However, you're printing it each time this check runs. Any reason?

Reply:
So, this statement should be there in each check right?  I mean all the checks should be consistent.